### ISSUE TITLE
Blocked on Implementing ValidationGroup and CauseValidation

### ISSUE THREAD

**Issue by Kahbazi** [FritzAndFriends/BlazorWebFormsComponents#36]
In order to implement `ValidationGroup` and `CauseValidation`, we need to get the element causing submit in `EditForm`, which is not possible in blazor right now. 
I opened an issue on AspNetCore repo requesting this feature. https://github.com/dotnet/aspnetcore/issues/18326

**Comment by csharpfritz** [comment]
I'll show this on stream Thursday the 16th and we can discuss alternate paths forward

**Comment by hishamco** [comment]
An updates on this one?

**Comment by csharpfritz** [comment]
Nothing yet...  will re-evaluate next week

**Comment by hishamco** [comment]
Duplicate of #247
